dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -702,6 +706,8 @@ CalpontSystemCatalog::OID CalpontSystemCatalog::lookupOID(const TableColName& ta
     }
     else if ((*it)->ColumnOID() == oid[15])
       ct.nextvalue = ((*it)->GetData(0));
+    else if ((*it)->ColumnOID() == oid[17])
```

Comment:
I am curious why does the if condition for the 17th element goes before the check for the 16th?

Reply:
Fixed.